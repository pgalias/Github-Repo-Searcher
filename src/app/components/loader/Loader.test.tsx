import React from 'react';
import { screen, render } from '@testing-library/react';
import { Loader } from './Loader';

describe('Loader component', () => {
  test('should render loader', () => {
    render(<Loader />);

    expect(screen.getByRole('alert')).toMatchSnapshot();
  });
});
