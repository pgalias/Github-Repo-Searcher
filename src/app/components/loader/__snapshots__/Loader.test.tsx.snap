// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render loader 1`] = `
<div
  aria-busy="true"
  class="css-q3dffh"
  role="alert"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
